src/Common/Hooks/useStateFilters/useStateFilters.tsx

```diff
@@ -64,10 +84,21 @@ const useStateFilters = <T = string,>({
         })
     }
 
+    const resetPagination = () => {
+        setPagination({
+            pageSize: DEFAULT_BASE_PAGE_SIZE,
+            offset: 0,
+        })
+    }
```

Comment:
- I think the resetPagination should not be exposed. Instead it should be called on any filter change. eg: handleSorting etc
- We should not be resetting the page size explicitly. On changing the filter we should change page number to 1